src/Microsoft.Extensions.Logging.Console/ConsoleLoggerProvider.cs

```diff
@@ -77,11 +86,14 @@ private void OnConfigurationReload(object state)
                 _settings = _settings.Reload();
 
                 _includeScopes = _settings?.IncludeScopes ?? false;
+                bool disableColors = _settings?.DisableColors ?? false;
```

Comment:
Can you use `var` here?

Reply:
addressed in 67c4ee557b7597425a7f722551cf4e5514a81df6